plugins/package-managers/spm/src/main/kotlin/SpmModel.kt

```diff
@@ -85,11 +85,11 @@ data class LibraryDependency(
 
     override val id: Identifier
         get() {
-            val (author, project) = parseAuthorAndProjectFromRepo(repositoryUrl)
+            val (author, _) = parseAuthorAndProjectFromRepo(repositoryUrl)
```

Comment:
IMO it's hard to understand why this is correct without looking up the implementation of `parseAuthorAndProjectFromRepo()`. What would have helped me during the review is a sentence like this in the commit message:

"The `parseAuthorAndProjectFromRepo()` function tries to parse the project name from the repository URL. However, this requires the URL to point to a known VCS host, and generally the dedicated `name` field from the SPM data model should be used instead."